servicepulse message details and menu organisation (#7098)
Particular/docs.particular.net: platform/servicepulse.include.md

ServicePulse is a web application designed for both administrators and developers. 

> #### For Administrators

Reviewer: It may be better to referring to the 2 different stake holders as Developers and Operations

Author: this text is taken from ServiceInsight documentation... system Administrators could belong to Operations, but that's making big assumptions about the customer's structure

> It provides a clear, near real-time, high-level overview of how a system is currently functioning. The UI also provides common failure recovery operations, such as retrying failed messages.

Reviewer: ```suggestion
It provides a clear, near-real-time, high-level overview of a system's current functioning. The UI also provides everyday failure recovery operations, such as retrying failed messages.
```

> ServicePulse provides a rich, graphical view of [detailed performance metrics](/monitoring/metrics/in-servicepulse.md). These are shown for logical endpoints, physical instances, and even individual message types.

Reviewer: ```suggestion
ServicePulse provides a rich, graphical view of [detailed performance metrics](/monitoring/metrics/in-servicepulse.md). These are shown for logical endpoints, physical instances, and individual message types.
```

> ServicePulse is required to generate [usage reports](/servicepulse/usage.md) for determining licensing requirements

Reviewer: ```suggestion
ServicePulse is required to generate [usage reports](/service pulse/usage.md) to determine licensing requirements.
```

#### For Developers

It provides advanced debugging, visualizations of messages flowing through the system, saga state changes, and more.

> It's quicker and easier to spot anomalies and incorrect behavior in a system using message flow diagrams, sequence diagrams, and other visualizations provided by ServicePulse. More detailed information, such as message headers and message metadata, is also shown.

Reviewer: ```suggestion
Using message flow diagrams, sequence diagrams, and other visualizations provided by ServicePulse, anomalies and incorrect behaviour in a system can be quickly and easily detected. More detailed information, such as message headers and message metadata, can also be shown.
```
